src/global.scss

```diff
@@ -15,3 +15,7 @@ html, body {
   z-index: 1000;
   min-width: 50px;
 }
+
+.numeric-readable {
+  font-family: 'Inconsolata', monospace;
+}
```

Comment:
Nice so the component is less specific to our use case 👍

Reply:
Exactly, that's my point :thinking: